src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -170,6 +160,17 @@ export class AddEditProjectComponent implements OnInit {
       });
     });
 
+    //This is to get Region infomation from List (db) and put into a list(regions)
+    this.config.lists.map(item => {
+      switch (item.type) {
+        case 'region':
+          this.temp.push(Object.assign({}, item));
+          break;
+      }
+    });
+    for(var _i = 0; _i<this.temp.length; _i++){
+      this.regions.push(this.temp[_i].name);
+    }
```

Comment:
This is an unecessary double loop.  Minimize to one.

Reply:
Hi Mark, I think I accidentally add Kathy's commit into this pr. I am still not really comfortable with git commit process. I will ask Max about what is going on here tomorrow. Sorry for create a mass here..